manager/assets/modext/core/modx.js

```diff
@@ -551,7 +551,6 @@ Ext.extend(MODx,Ext.Component,{
             });
             tabPanel.add(newTabConfig);
             tabPanel.doLayout();
-            tabPanel.setActiveTab(0);
```

Comment:
Ohhhhh this must be why VersionX always kept forcing the first tab to open, breaking the functionality that keeps tabs open. Never figured that out, but that does use this method 😂

Reply:
Yes, I was kinda pulling my hair out for a while there because last I worked on this all was well. But in between I did install VersionX and after digging found that this line was causing havoc. I poked around to see if it was really needed, and found it was not.